runtime/src/append_vec.rs

```diff
@@ -21,6 +21,8 @@ macro_rules! align_up {
     };
 }
 
+const MAXIMUM_APPEND_VEC_FILE_SIZE: usize = 16 * 1024 * 1024 * 1024; // 16 GiB
```

Comment:
Could be much lower considering it seems AccountsDB never increases its `.file_size` from `DEFAULT_FILE_SIZE`.

Reply:
It doesn't increase, it may allocate a larger one than default if the store account is larger than the default file size.

Reply:
Thanks for confirmation, then maybe we could lower to be close in relation to #7320.

Reply:
For now, I think this is fine as it is now because we can assume the running system will have at least 16GB RAM as per [the doc](https://docs.solana.com/book/running-validator/validator-hardware#recommended-setups) and we can be relieved we'll never be tripped by the limit at that high number depending on current AccountDB's behavior and its possible future changes.

Or, could this be lower, say, a half?: 8GB